tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -550,15 +552,16 @@ func TestRateLimit(t *testing.T) {
 	got, err := vectorValue(value, map[string]string{responseCodeLabel: "429", "destination_version": "v1"})
 	if err != nil {
 		t.Logf("prometheus values for request_count:\n%s", promDump(promAPI, "request_count"))
-		fatalf(t, "Could not find rate limit value: %v", err)
+		errorf(t, "Could not find 429s: %v", err)
+		got = 0 // want to see 200 rate even if a bug makes there is no 429s
```

Comment:
Can you reword the comment.
`want to see 200 rate even if no 429s were recorded`

Reply:
changed